x/staking/keeper/slash.go

```diff
@@ -21,7 +21,7 @@ import (
 // CONTRACT:
 //    Infraction was committed at the current height or at a past height,
 //    not at a height in the future
-func (k Keeper) Slash(ctx sdk.Context, consAddr sdk.ConsAddress, infractionHeight int64, power int64, slashFactor sdk.Dec) {
+func (k Keeper) Slash(ctx sdk.Context, consAddr sdk.ConsAddress, infractionHeight int64, power int64, slashFactor sdk.Dec) sdk.Int {
```

Comment:
Could you add this to the API breaking change section too?

Reply:
[added](https://github.com/cosmos/cosmos-sdk/pull/9458/commits/6a2da56737e7dbab9a926be626e511b6e1506e74)